src/test/java/school/hei/haapi/integration/conf/TestUtils.java

```diff
@@ -821,13 +821,13 @@ public static Comment comment1() {
         .creationDatetime(Instant.parse("2021-11-09T08:26:24.00Z"));
   }
 
-  public static Comment comment2() {
+  public static Comment excpectedComment2() {
```

Comment:
you didn't really need to rename the function name comment2() to expectedComment2()